src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1057,6 +1058,16 @@ constexpr uint64_t kMemoryWarningThresholdKbDefaultValue = 1024 * 1024 * 8;  //
 
 void OrbitMainWindow::LoadCaptureOptionsIntoApp() {
   QSettings settings;
+  if (settings.value(kEnableSamplingSettingKey, true).toBool()) {
+    bool ok = false;
+    double sampling_period_ms = settings.value(kSamplingPeriodMsSettingKey, 1.0).toDouble(&ok);
+    CHECK(ok);
+    CHECK(sampling_period_ms > 0.0);
```

Comment:
I would prefer to not make this a `CHECK` as the value is read from outside of Orbit and could thus be modified outside of our control to make this check fail. IMO it's better to reset the value to be within the predefined bounds in this case.

Reply:
Yep, done, see other answer.